apps/zoom/.eslintrc.json

```
@@ -0,0 +1,3 @@
+{
+  "extends": "next/core-web-vitals"
```

```suggestion
  "extends": "'custom/next'"
```

The project eslint configuration should be used here. If it's not working, you can refer to the `github` branch configuration. We might have to update the template.